lib/src/common/variables.ts

```diff
@@ -506,6 +506,7 @@ export const componentTokens = {
     selectedIconColor: CoreTokens.color_grey_700,
     unselectedIconColor: CoreTokens.color_grey_700,
     disabledIconColor: CoreTokens.color_grey_500,
+    dividerThickness: "2px",
```

Comment:
Has this token been agreed with a designer?

Reply:
It has been decided that this token is not needed, so I am removing it.